pint_server/app.py

```diff
@@ -16,13 +16,23 @@
 # you may find current contact information at www.suse.com
 
 import datetime
+import math
 import re
 from decimal import Decimal
-from flask import (abort, Flask, jsonify, make_response, request, redirect,
-                   Response)
+from flask import (
+    abort,
+    Flask,
+    jsonify,
+    make_response,
```

Comment:
FYI just noticed that we are importing a make_response from flask here, but further down we define a make_response at line 501...

Reply:
It seems to me the import of `make_response` from Flask is not neeeded ?

Reply:
good catch!